geofencing-spec.html

```diff
@@ -93,32 +93,29 @@
       </p>
       <pre class="example highlight" title="Monitor a region">
 // https://example.com/webapp.js
-navigator.serviceWorker.register('serviceworker.js').then(
-  function(serviceWorkerRegistration) {
-    serviceWorkerRegistration.geofencing.add(
-        new CircularGeofenceRegion({
-          name: "myfence",
-          latitude: 37.421999,
-          longitude: -122.084015,
-          radius: 1000
-        }), {includePosition: true}).then(
-      function(geofence) {
-        console.log(geofence.id);
-        // If more than just a name needs to be stored with a geofence, now
```

Comment:
I think it would be good if somehow this comment could be kept, as it kind of explains what the ID is good for